src/main/resources/locale_pt_PT.properties

```
@@ -0,0 +1,196 @@
+command.repSettings.sub.mention=Get or set whether mention messages should be analyzed.
+command.repSettings.sub.mention.false=Bot não pesquisará nas menções.
+command.repSettings.sub.mention.true=Bot irá pesquisar em menções.
+command.repSettings.sub.minMessages=O utilizador pode sempre agradecer aos autores destas mensagens.
+command.repSettings.sub.minMessages.get=O utilizador pode sempre agradecer aos autores das últimas mensagens %AMOUNT%.
+command.repSettings.sub.reaction=Apanhar ou definir o emote ou emoji de reputação.
+command.repSettings.sub.reaction.get.emoji=Reputação actual Emoji está definido para: %EMOJI%
+command.repSettings.sub.reaction.get.emote=Reputação actual Emote está definido para: %EMOTE%
+command.repSettings.sub.reaction.get.error=Não poderia encontrar emoção na guilda.
+command.repSettings.sub.reaction.set.emoji=Reputação Emoji set to %EMOJI%
+command.repSettings.sub.reaction.set.emote=Reputation Emote set to %EMOTE%
+command.repSettings.sub.reactions=Get or set whether reputation reactions should give reputation.
+command.repSettings.sub.reactions.false=Bot não irá processar reacções.
+command.repSettings.sub.reactions.true=Bot irá processar reacções.
+command.reputation.description=Informação sobre a sua reputação ou sobre a reputação de outros.
+command.reputation.profile.nextLevel=Próximo nível
+command.reputation.profile.title=Reputação de %NAME%
+command.reputation.sub.top=Abter a lista do topo ou uma página da lista
+command.reputation.sub.top.ranking=Ranking de %GUILD%
+command.roles.description=Gerir papéis de reputação.
+command.roles.sub.add=Adicionar um papel de reputação.
+command.roles.sub.add.added=Added role %ROLE% com reputação %POINTS%.
+command.roles.sub.list=Listar papéis.de reputação.
+command.roles.sub.managerRole=Alterar o papel de gerente. Os utilizadores com esta função podem gerir o bot.
+command.roles.sub.managerRole.current=A actual função de gestor é %ROLE%.
+command.roles.sub.managerRole.noRole=Sem função de gestor definida.
+command.roles.sub.managerRole.set=A função de gestor definida como %ROLE%.
+command.roles.sub.remove=Remover uma função de reputação.
+command.roles.sub.remove.notARepRole=Esta função não é uma função de reputação.
+command.roles.sub.remove.removed=Remover um papel %ROLE%.
+command.scan.canceled=Scan cancelado.
+command.scan.canceling=Tentativa de cancelar o scan.
+command.scan.completed=Varredura concluída.
+command.scan.description=Scan a channel for reputations.
+command.scan.error.history=Desaparecimento da permissão para histórico de mensagens
+command.scan.error.queueFull=Todos os scans estão actualmente em execução. Por favor aguarde até que uma ranhura esteja livre.
+command.scan.error.running=Já está a decorrer uma tarefa nesta guild. Por favor aguarde até que a tarefa esteja terminada.
+command.scan.progress=Progress: %PERCENT%
+command.scan.result=Scanned %SCANNED% messages.\nAdded %HITS% reputation.
```

Looks untranslated

DeepL suggestion:
```suggestion
command.scan.result=Mensagens digitalizadas %SCANNED%.\nAdicionado %HITS% reputação.
```